website_form_recaptcha/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,12 @@
+First of all you must obtain
+a ReCaptcha key from `Google <http://www.google.com/recaptcha/admin>`_
+
+**Global setup**
```

Comment:
I have a question. Does having a global setup make sense in any circumstance? Or is it just to keep backwards compatibility? :thinking: 

If it is the 2nd, then maybe you should add a migration script and we could forget about the global system now and forever.

Reply:
I think in multi-website you can easily have same captcha credentials set up, so I don't think this is an issue

Reply:
nope, you can easily have different to domains. And this is also the answer for @Yajo 's question:

1. if you have N websites w/ same domain -> use the param
2. if you have N websites w/ different domain -> use website settings
3. it offers backward compat for free

Reply:
well, AFAIS you can set a domain and this will be applied to sub-domains too.
If you set a sub-domain, it works on it only.

Reply:
Oh ok, thanks for the info!